README.Rmd

```diff
@@ -113,7 +113,7 @@ in the optimal trait value of one variable which results from a one median
 absolute deviation increase in another variable.
 
 ```{r authority-delta-theta, fig.height=5, fig.width=5}
-coev_plot_delta_theta(fit)
+coev_plot_delta_theta(fit, limits = c(-5, 50))
```

Comment:
Why such a large positive limit? To show that they keep increasing even at large values? Is this now far beyond the sample range of the latent trait?